packages/now-build-utils/src/fs/run-user-scripts.ts

```diff
@@ -69,10 +69,19 @@ export function getSpawnOptions(
   return opts;
 }
 
-export async function getNodeVersion(destPath: string): Promise<NodeVersion> {
+export async function getNodeVersion(destPath: string, minNodeVersion?: string): Promise<NodeVersion> {
   const { packageJson } = await scanParentDirs(destPath, true);
   const range = packageJson && packageJson.engines && packageJson.engines.node;
```

Comment:
```suggestion
  const range = (packageJson && packageJson.engines && packageJson.engines.node) || minNodeVersion;
  return getSupportedNodeVersion(range);
```

Now you can change this line 👍